src/test/regress/expected/multi_insert_select.out

```diff
@@ -1475,3 +1475,238 @@ DEBUG:  ProcessUtility
 CREATE VIEW test_view AS SELECT * FROM raw_events_first;
 INSERT INTO raw_events_second SELECT * FROM test_view;
 ERROR:  cannot plan queries that include both regular and partitioned relations
+-- we need this in our next test
+truncate raw_events_first;
+SET client_min_messages TO DEBUG4;
```

Comment:
We don't have to, but, I find it useful so that we also can see which worker queries are (or are not) generated for per interval.

Do you mean decreasing the level to `DEBUG2` or so? I used `DEBUG4` since our worker queries are shown with it in other places.